bin/offline.cpp

```diff
@@ -136,15 +142,37 @@ int main(int argc, char *argv[]) {
     fileSource.setAccessToken(token);
     fileSource.setAPIBaseURL(apiBaseURL);
 
+    if (inputDb && mergePath) {
```

Comment:
While this is not a high priority tool, can we add a small error message to let people know if they've only supplied one, but not both parameters?

Reply:
Addressed by using an `ArgumentsGroup`